merp_outgoing_routing/models/res_company.py

```diff
@@ -9,15 +9,24 @@ class Company(models.Model):
 
     outgoing_routing_strategy = fields.Selection(
         [
-            ('name', 'Sort by source locations in alphabetical order'),
-            ('removal_prio', 'Sort by location removal strategy priority'),
-            ('product', 'Sort by product name'),
+            # path should be valid for both stock pickings and quants
+            ('location_id.removal_prio', 'Location removal priority'),
+            ('location_id.name', 'Location name'),
+            ('product_id.name', 'Product name'),
```

Comment:
I think  merp_picking_list_sorted should be part of this module. Can you merge it into this and delete standalone module?

Reply:
merged into `merp_outgoing_routing` and removed